src/discljord/cdn.clj

```diff
@@ -0,0 +1,72 @@
+(ns discljord.cdn
+  "Namespace with functions to create [cdn urls](https://discord.com/developers/docs/reference#image-formatting) to image data from API entities such as users or guilds."
+  (:require [clojure.string :refer [starts-with?]]))
+
+(def base-url "https://cdn.discordapp.com")
+
+(defn custom-emoji
+  "Takes a custom emoji object or id and returns a url to the image data.
+  Note that an id alone is not enough to determine whether the emoji is animated or not,
+  so if you pass an id you will always get a url to a png."
```

Comment:
For consistency with the rest of my docstrings, anything after the initial short description should be separated by a blank line.

Reply:
By "initial short description" do you just mean the first line/sentence?

Reply:
Yes, there's some parsers for the docstrings which use the first line as a short description and the rest is treated separately.